### ISSUE TITLE
Storage Access

### ISSUE THREAD

**Issue by ramuvr** [SharePoint/sp-dev-docs#9802]
When Microsoft provides the Storage account, is the traffic private within Microsoft network between the API and Storage?

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: e94f5162-1e54-2ce1-238a-20fb96d4093c
* Version Independent ID: 1a1f8f24-df75-7478-6cd5-cf612daf77f8
* Content: [SPO provided Migration Azure container and queue](https://learn.microsoft.com/en-us/sharepoint/dev/apis/migration-api-azure-container-and-queue)
* Content Source: [docs/apis/migration-api-azure-container-and-queue.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/apis/migration-api-azure-container-and-queue.md)
* Service: **sharepoint-online**
* Sub-service: **migration-tool**
* GitHub Login: @JoanneHendrickson
* Microsoft Alias: **jhendr**

**Comment by Ashlesha-MSFT** [comment]
Hello @ramuvr,
Thank you for bringing this issue to our attention. We will look into it and get back to you shortly.

**Comment by Ashlesha-MSFT** [comment]
@ramuvr,
Yes, if configured properly, traffic between an Azure service (such as your API) and an Azure Storage account can stay entirely within Microsoft’s private network and never traverse the public internet.
 [Learn more about Azure Storage Private Endpoints](https://learn.microsoft.com/en-us/azure/storage/common/storage-private-endpoints) — explains how Private Endpoints work and how they ensure secure, internal traffic routing.

**Comment by ramuvr** [comment]
@Ashlesha-MSFT - the question is whether the "default container while using the migration API." referred to in the article, also uses private connectivity within Microsoft (not when customer uses their own Storage account)

**Comment by Ashlesha-MSFT** [comment]
@ramuvr,
As of now, the official documentation does not explicitly confirm whether this traffic remains within Microsoft’s private network. While it's likely that the communication happens internally based on Microsoft’s infrastructure design, this is not clearly documented.

To get this clarified and possibly included in future documentation, I recommend raising a feature request for a documentation enhancement. This will help ensure that Microsoft provides an authoritative statement on this behavior, which is especially important for environments with strict compliance and security requirements.

You can submit the feature request here: (https://feedbackportal.microsoft.com/feedback/forum/06735c62-321c-ec11-b6e7-0022481f8472)

**Comment by ramuvr** [comment]
Its not really a feature that I am requesting, just asking the Documentation to be updated. I don't think its appropriate to raise a feature request for updating documentation. Could you pass this to someone from engineering who can confirm and update a note on the documentation ?

**Comment by Ashlesha-MSFT** [comment]
@ramuvr,
Thanks for the clarification — apologies, I now understand that this falls under a documentation update rather than a feature request.

I'll go ahead and pass this along to the engineering team for further review.
